packages/ui/src/components/NotebookPost/NotebookPost.tsx

```diff
@@ -67,15 +64,14 @@ export default function NotebookPost({
             }}
             width="100%"
             height={smallImage ? IMAGE_HEIGHT / 2 : IMAGE_HEIGHT}
-            borderRadius="$m"
+            borderRadius="$s"
           />
         )}
         {post.title && (
           <Text
+            fontWeight={'500'}
```

Comment:
Should use one of the tamagui tokens here, and if we don't have one maybe we should add one.